sdk/keyvault/azure-keyvault-administration/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features Added
 - Added support for service API version `7.6-preview.1`
```

Comment:
Note that `7.6-preview.1` support (which just involved updating our generated code, API version enum, and changelogs) is reverted in a separate PR to keep this diff simple: https://github.com/Azure/azure-sdk-for-python/pull/37827